disqus/public/class-disqus-public.php

```diff
@@ -300,6 +300,11 @@ private function dsq_embed_can_load_for_post( $post ) {
 		if ( 'open' != $post->comment_status ) {
```

Comment:
It looks like `comments_open()` includes this check along with a filter, can you just replace this check with `comments_open()` instead of adding the new one below?

Reply:
I don't really know whether we can just replace instead of adding. I think in most cases these two conditions are the same, but who knows what plugin can change one of them...

Reply:
OK I can accept that, it won't hurt.